qa/L0_python-self-test/test.sh

```diff
@@ -12,6 +12,9 @@ test_body() {
     nosetests --verbose test_pipeline.py
     nosetests --verbose test_decoders.py
     nosetests --verbose test_python_function_operator.py
+    for test_script in $(ls test_operator_*.py); do
+        nosetests --verbose ${test_script}
```

Comment:
I think you can rename `dali/test/python/test_decoders.py` to `dali/test/python/test_operator_decoder.py`

Reply:
addressed in #930